### PR TITLE
Update some more of the cms imports to use local-dev-lib

### PR DESCRIPTION
**Pull request by brandenrodgers**
## Description and Context

Updating imports for `cli-lib/functions`, `cli-lib/modules`, and `cli-lib/templates` to pull from their new local-dev-lib counterparts.

### CODE REVIEW

**Comment by brandenrodgers** [comment]
This is ready for review now, although it's dependent on the latest changes over in local-dev-lib. We will need to cut a new patch release of LDL before merging/releasing this.

**Review by camden11 — APPROVED** [review]
Tested and works well! Should be good to merge once LDL update is out

File: packages/cli/lib/commonOpts.js
Comment (brandenrodgers) [inline packages/cli/lib/commonOpts.js:75]: I realized that certain log statements in local-dev-lib were not working correctly. It was because the default log level is `NONE`, and we were only setting the log level to `LOG` for the cli-lib instance of the logger.

**Review by camden11 — APPROVED** [review]

**Comment by camden11** [comment]
Good catch with the log level stuff. Also this whole PR is a pretty solid case against log callbacks. Looks like there are some tests failing but otherwise looks good

**Comment by brandenrodgers** [comment]
I updated everything that uses logCallbacks in response to: https://github.com/HubSpot/hubspot-local-dev-lib/pull/95

**Review by camden11 — APPROVED** [review]
Log callback stuff looks good!

**Pull request merged**
